src/features/pages/home/p2p-banner/index.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import { StaticImage } from 'gatsby-plugin-image'
+import { p2p_box_container, p2p_qr_container } from './styles.module.scss'
+import Container from 'features/components/atoms/container'
+import Box from 'features/components/atoms/box'
+import FlexBox from 'features/components/atoms/flex-box'
+import Typography from 'features/components/atoms/typography'
+import Link from 'features/components/atoms/link'
+import { Localize } from 'components/localization'
+import dclsx from 'features/utils/dclsx'
+
+const P2PBanner = () => {
+    return (
+        <Container.Fixed as="section" bgcolor="tertiary">
+            <FlexBox
+                container="fixed"
+                direction="col-reverse"
+                md={{ direction: 'row', justify: 'between' }}
+            >
+                <Box mt="9x" md={{ mt: '0x' }}>
+                    <StaticImage
+                        src="../../../../images/common/home/p2p_home_banner.png"
+                        alt="p2p banner"
+                        height={400}
+                        objectFit="cover"
+                        placeholder="none"
+                        formats={['avif', 'webp', 'auto']}
+                    />
+                </Box>
+                <FlexBox
+                    align="center"
+                    gap="12x"
+                    padding_inline="8x"
+                    padding_block="40x"
+                    md={{ mr: '10x', padding: '0x' }}
+                    lg={{ mr: '40x' }}
+                >
+                    <FlexBox className={p2p_box_container} gap="8x" direction="col">
+                        <StaticImage
+                            src="../../../../images/common/home/deriv_p2p.png"
```

Comment:
use should handle the RTL image as well, if needed

Reply:
How can i do that?

Reply:
I think there is another image for RTL, you can use that.